fdbclient/include/fdbclient/TenantManagement.actor.h

```diff
@@ -145,6 +145,7 @@ Future<std::pair<Optional<TenantMapEntry>, bool>> createTenantTransaction(
 		throw invalid_tenant_group_name();
 	}
 
+	tenantEntry.tenantName = name;
```

Comment:
An upcoming PR is going to make it so that `name` isn't even an argument to this function and the name will need to be set on the tenant entry instead, but to keep things simple here I just assign it into the tenant entry.